### PR TITLE
ET-3691 script for running loadtests

### PR DESCRIPTION
**Pull request by Robert-Steiner**
## What 🕵️ 🔍

- adds two just commands for setting up/down a loadtest
- adds and upload test data script
- changes the tenant config from `hcl` to `json`. this way we can use jq to change the tenant name
- adds some more test classes (now they are using the real document ids from the test data)

TODO
- [x] upload test data to s3 bucket
- [x] script that ingests data

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/ET-3691)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.
  *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *

### CODE REVIEW

**Review by jorge-cr-13 — DISMISSED** [review]
LGTM! :)

File: aws/b2b_personalization/_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- aws cli
+
+Login (aws sso login --profile <profile>) with your `AdministratorAccess-917039226361`
+and `AdministratorAccess-165060789769` profile first before starting any of the following commands.
+
+## Run locust
+
+```shell
+USER_API_KEY=<api key> just run-locust <api endpoint> <additional arguments>
+```
```
Comment (joecks) [inline aws/b2b_personalization/_tests/README.md:16]: The only small thing here is that this command at this point is a bit confusing in the README. 
Maybe a quickstart section would be nice: 


## Quick start

```shell
aws sso login  --profile AdministratorAccess-917039226361
aws sso login  --profile AdministratorAccess-165060789769

just loadtest-new
just loadtest-run
# check http://0.0.0.0:8089/
just loadtest-data-download
just loadtest-destroy
```

**Review by joecks — APPROVED** [review]

File: aws/b2b_personalization/_tests/loadtest/.gitignore [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+.pybuilder/
+target/
+
+# Jupyter Notebook
+.ipynb_checkpoints
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply aws/b2b_personalization/_tests/loadtest/.gitignore:79]: :D i have it from here https://github.com/github/gitignore

File: aws/b2b_personalization/_tests/README.md (hunk 1 above)
Comment (Robert-Steiner) [reply aws/b2b_personalization/_tests/README.md:30]: I have added it to the readme

File: aws/b2b_personalization/_tests/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Download load test data
+
+```shell
+just loadtest-data-download
+```
+
+## Upload load test data
+
+```shell
+DOCUMENTS_API_KEY=<api key> just loadtest-data-upload <input> <api endpoint>
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply aws/b2b_personalization/_tests/README.md]: I will create a ticket

**Review by joecks — APPROVED** [review]

**Pull request merged**